drivers/ft5x06/include/ft5x06_params.h

```diff
@@ -49,13 +49,17 @@ extern "C" {
 #ifndef FT5X06_PARAM_YMAX
 #define FT5X06_PARAM_YMAX               (272U)
 #endif
+#ifndef FT5X06_PARAM_TYPE
+#define FT5X06_PARAM_TYPE               FT5X06_TYPE_FT5336
```

Comment:
This is missing documentation. Other defines are added in the [doccheck excluded patterns list](https://github.com/RIOT-OS/RIOT/blob/master/dist/tools/doccheck/exclude_patterns).

Reply:
Thanks, added in cd30be5b8b40cb0dcfee37297e26682ef70747a8